src/mono/browser/build/WasmApp.InTree.props

```diff
@@ -15,6 +15,7 @@
     <RunAOTCompilation Condition="'$(RunAOTCompilation)' == ''">false</RunAOTCompilation>
     <PublishTrimmed>true</PublishTrimmed>
     <RunAnalyzers>false</RunAnalyzers>
+    <FeaturePerfTracing>true</FeaturePerfTracing>
```

Comment:
Does it do any harm to wasi? https://github.com/dotnet/runtime/blob/main/src/mono/sample/wasm/Directory.Build.props#L39

Reply:
This is browser only, there is also `WasiApp.InTree.props`
Maybe we can make it less confusing later.